README.md

```diff
@@ -15,8 +15,10 @@ Rear Rider is an alert system to notify cyclists of approaching objects and vehi
 
 ## Running main Rear Rider backend application.
 
+Make sure the Raspberry Pi is configured to work as an access point on wlan0. The [offical guide from Raspberry Pi](https://www.raspberrypi.com/documentation/computers/configuration.html#setting-up-a-routed-wireless-access-point) has excellent step-by-step instructions.
```

Comment:
Nice! I didn't even think to look at the OFFICIAL docs for this haha. It would cool to include sample configuration that can be installed by a handcrafted script in the future to ensure everyone's device has the same configuration. This can be a future task.

Reply:
yeah, a script to handle all the config stuff would be nice. definitely a future me problem 😂